### PR TITLE
Update README document.

### PR DESCRIPTION
**Pull request by Hsiangkai**
Use pytest instead of nose to do unit tests. In addition, add package pytest-cov to do code coverage through pytest.

### CODE REVIEW

**Comment by flit** [comment]
@Hsiangkai Fyi, I'm also working on improving the pyOCD docs on this branch: https://github.com/flit/pyOCD/tree/dev_docs

**Comment by Hsiangkai** [comment]
Sorry, I missed it. I think you could provide more thoroughly update. You could ignore this pull request.

**Comment by flit** [comment]
Well, I'm just getting started with my changes. It will be a while before I finish. So it's probably a good idea to merge your changes first. I just wanted to let you know what was coming eventually.

File: dev-requirements.txt [hunk 1]
```diff
@@ -3,5 +3,4 @@
 
 pytest
 pytest-catchlog
-coverage
-elapsedtimer
```
Comment (c1728p9) [inline dev-requirements.txt]: I think elapsedtimer is used for some tests

**Comment by Hsiangkai** [comment]
@c1728p9 Thanks for your reminder. I did not notice Travis is failed. I already corrected the patch.

**Comment by flit** [comment]
Closing this issue. The readme and docs were updated in #427.

**Pull request closed without merging**